arroyo-api/src/jobs.rs

```diff
@@ -368,6 +377,89 @@ pub async fn get_job_checkpoints(
     }))
 }
 
+/// Subscribe to a job's output
+#[utoipa::path(
+    get,
+    path = "/v1/pipelines/{pipeline_id}/jobs/{job_id}/stream",
```

Comment:
what do you think about a less generic name than stream? maybe output?

Reply:
Sure, done.